### ISSUE TITLE
App review comments DB No. 9

### ISSUE THREAD

**Issue by phillybroadbent** [webappsforengineers/wa4e-v2#36]
Reference: Mudmat UU & CU VH2M2T 

In the example problem considered application of torsion seem to yield higher horizontal and moment capacity?  

Notes: The ‘further reduced due to T’ are bigger than the values above … think labels just wrong way round? – check against original http://www.webappsforengineers.com/vh2m2t/index.html

**Comment by phillybroadbent** [comment]
This is the same in the original

**Comment by phillybroadbent** [comment]
Susan is checking whether the values in 'further reduced due to T' are correct.

**Comment by phillybroadbent** [comment]
The envelopes plotted (in ss or in apps) are those in ‘In direction of loading Hmax and Mmax’ and ‘Further reduced values due to T Hmax and Mmax’. The shape of the envelope is constant and just scaled by these apex points.

In UU CU mudmat, I think the numbers in the two sets of boxes are just the wrong way round – compare with cells Z25 and 26 and Z32 and 33 in the attached excel calculation spreadheet – the values reduced due to Torsion should always be lower.

This is confirmed if you increase the size of one of the eccentricities – e.g. ezHx from 4 to 6, increasing Torque from 280 to 480, the output values in the web app for ‘in direction of loading’ reduce, but the ones reduced for Torsion remain the same (and should be other way round) – and can see the right way round in the spreadsheet.

So - the outer envelope should always be blue and the inner one always orange – which is correct in the hybrid mudmat/pinpiles app with ‘mat only’ toggled – and the series are named Zero Torque for blue, and T = some value for orange. Could we also have the same labels for UU CU, rather than trace 0 and 1? I think the colour/series connections are probably right – and when the right values are in the right cells in the web app, the orange series will be inside the blue one.

Also to note in the UU CU app, that when using Tlag (time) = 0, which is ‘UU’ in this app, this should give the same answers as in the ‘hybrid mudmat’ app when selecting ‘mat only’, and also in the UTI/ZTI mudmat app for the UTI conditions. i.e. if you put the same values for input in the UTI/ZTI app, with UTI toggled – then the output is the same (but in the correct boxes for ‘in direction of loading’ and ‘reduced due to T’). It would be good if we could have the same default input parameters in each of those apps

**Comment by phillybroadbent** [comment]
I've switched the values of 'in direction of loading' and 'further reduced values due to T' because these were previously the wrong way around.  The labels of the plot legend were the wrong was around and this has also been corrected.  In the wa4e-math vh2m2t script, the results values for zero torque and torque were also switched so this has been corrected as well.  For the input parameters common between the UU CC, UTI/ZTI, and hybrid mudmat apps, these have been set to the same default, using the values from UU CU.
